hooks/configure

```diff
@@ -74,6 +74,24 @@ switch_service() {
 	esac
 }
 
+update_pi_config_value() {
+    path="$1"
+    k="$2"
+    v="$3"
+
+    if [ -z "$v" ]; then
+        # empty - unset an option
+        sed -i -e "/^[ \t#]*${k}=/ s/^#*/#/" "$path"
```

Comment:
This is intentional - mostly because the config.txt looks like:
```
# uncomment to force a console size. By default it will be display's size minus
# overscan.
#framebuffer_width=1280
#framebuffer_height=720
```
So enable would uncomment, disable comment again without moving them in the file (unless the options are missing in which case things just get appended/commented)

But I do understand that it makes things more complicated so I'm happy to simplify as suggested (if desired).